xml/cap_depl_about.xml

```diff
@@ -39,7 +39,45 @@
 
   <para>
    &productname; is a software platform for cloud-native application deployment
-   based on &scf; and &kube;. It serves different but complementary purposes
+   based on &scf; and &kube;.
+</para>
+   
+ <para>
+&productname; describes the complete software stack, including the operating 
+system, &kube;, and &scf;. 
+  </para>
+   <para>
+&scf; is comprised of the &sle; builds of the UAA (User Account and
+Authentication) server, &scf;, the Stratos Web user interface, and Stratos Metrics (scheduled for the &scf; 1.3 release).
+</para>
+
+<para>
+  The &cf; code base provides the basic functionality. &scf; differentiates
+  itself from other &cf; distributions by running in Linux containers managed
+  by &kube;, rather than virtual machines managed with &bosh;, for greater
+  fault tolerance and lower memory use.
+ </para>
+
+<para>
+    All Docker images for the &sle; builds are hosted on registry.suse.com.
```

Comment:
`registry.suse.com` could use some formatting love

Reply:
I added literal tags-- it does not resolve, so I don't think it should be clickable.

Reply:
Works for me. Since it's a docker registry it doesn't resolve in a browser unless additional tools are used.